src/applications/medical-copays/containers/MedicalCopaysApp.jsx

```diff
@@ -27,7 +27,8 @@ const MedicalCopaysApp = ({ children }) => {
   useEffect(
     () => {
       if (userLoggedIn) {
-        dispatch(getStatements());
+        const generateGetStatements = () => getStatements(dispatch);
+        dispatch(generateGetStatements);
```

Comment:
You can also change this to:
```
dispatch(getStatements);
```

Reply:
You are absolutely right, thank you for catching that as well. I've just updated the same and confirm that the tests continue to pass after refactor.